metrics/metrics.go

```diff
@@ -0,0 +1,66 @@
+package metrics
+
+import (
+	"encoding/json"
+	"io"
+	"strings"
+
+	metrics "github.com/rcrowley/go-metrics"
+)
+
+const (
+	GRPC = "grpc"
+	SSE  = "sse"
+)
+
+type Metrics struct {
+	Type     string
+	registry metrics.Registry
+}
+
+const (
+	clientCounterName = "clients"
+)
+
+var registry metrics.Registry
+
+func init() {
+	registry = metrics.NewRegistry()
+}
+
+type Registry metrics.Registry
+
+func New(metricsType string) (Metrics, error) {
+	m := Metrics{
+		registry: registry,
+		Type:     metricsType,
+	}
+	return m, nil
+}
+
+func GetRegistry() metrics.Registry {
+	return registry
+}
+
+func (m *Metrics) getFieldName(names []string) string {
+	return strings.Join(append([]string{m.Type}, names...), ":")
+}
+
+func (m *Metrics) IncClientCount() {
```

Comment:
__[golint]__
* [golint] exported method Metrics.IncClientCount should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->